web/auth/auth.go

```diff
@@ -323,6 +326,22 @@ func AccessControl(logger *zap.Logger, db database.Database, scms *Scms) echo.Mi
 				return echo.NewHTTPError(http.StatusBadRequest, err)
 			}
 
+			token, err := c.Cookie("session")
+			if err != nil {
+				return err
+			}
+
+			// Check if user exists in map with an old session token
```

Comment:
This code could be broken out as 
- a function that takes `token` as input or 
- a method that operates on a struct that encapsulates the `TokenToUserMap` map

The latter version may require some additional refactoring. The next comments may give more clues which is better.

Ideally, we should avoid exporting a map directly like this. It would be better to use functions in the `auth` package to control the map accesses.